layouts/_default/baseof.html

```diff
@@ -82,8 +82,9 @@
     />
 
     {{ $js := resources.Match "js/*.js" }} {{ range $js }}
-    <script type="text/javascript" src="{{ .RelPermalink }}" defer></script>
+    <script type="module" src="{{ .RelPermalink }}" defer></script>
```

Comment:
https://caniuse.com/es6-module

Reply:
that's fine, it looks well supported. Is it just the new standard way of linking js or do we get something extra from it too?

Reply:
This is the new standard -- it means we are able to ship a smaller bundle that does not have a lot of compatibility code for old browsers. Parcel is generating this newer smaller bundle.

There is also a setting to generate an old, larger bundle with compatibility code, but I spent a ton of time trying to get both of them copied over and correctly inlined into the hugo template, and I gave up given this will probably impact 2% of users on browsers like IE....